.github/workflows/check.yml

```diff
@@ -57,6 +57,16 @@ jobs:
       - name: Run Tests
         run: bundle exec fastlane ${{ matrix.lane }}
       
+      - name: Check Static Analyzer Output
+        run:  |
+          if [[ -z `find clang -name "*.html"` ]]; then
+            echo "Static Analyzer found no issues. Fail for demo purpose."
+            exit 123
+          else
+            echo "Static Analyzer found some issues"
+            exit 1234
```

Comment:
I'm not sure anything but a value of `1` adds any value here. `1234` suggests meaning that doesn't exist.

Reply:
Sorry, it is a draft, I've just forgotten to convert it. I've picked these values randomly (possibly avoiding overlap with something else). `exit` in both paths of `if else` was for test purpose only.

Reply:
806e5c9ad6c1138bf0e3a98422695348b6574db7